main.go

```diff
@@ -122,7 +122,10 @@ func main() {
 	fs.IntVar(&cli.SerfEventBuffer, "serf-event-buffer", 100000, "Size of serf 'recent event' buffer, outside of which things are dropped")
 	fs.IntVar(&cli.SerfMaxQueueDepth, "serf-max-queue-depth", 100000, "Size of Serf queue, outside of which things are dropped")
 	fs.StringVar(&cli.EnableAnalytics, "analytics", "disabled", "Enables analytics API: enabled or disabled")
-	fs.StringVar(&cli.AnalyticsMetricsURL, "analytics-metrics-url", "", "URL of Prometheus DB to store analytics data")
+	fs.StringVar(&cli.KafkaBootstrapServers, "kafka-bootstrap-servers", "", "URL of Kafka Bootstrap Servers")
+	fs.StringVar(&cli.KafkaUser, "kafka-user", "", "Kafka Username")
+	fs.StringVar(&cli.KafkaPassword, "kafka-password", "", "Kafka Password")
+	fs.StringVar(&cli.KafkaTopic, "kafka-topic", "", "Kafka Topic used to send analytics logs")
```

Comment:
Should this be more specific in case we have other topics in the future? e.g.
```suggestion
	fs.StringVar(&cli.AnalyticsKafkaTopic, "analytics-kafka-topic", "", "Kafka Topic used to send analytics logs")
```